app/controllers/concerns/consentable.rb

```diff
@@ -1,30 +0,0 @@
-module Consentable
-  extend ActiveSupport::Concern
-
-  included do
-    before_action :set_consented_at, only: [:create]
-  end
-
-  def consent_check_passed?
-    return true unless consent_applicable?
-    consented?
-  end
-
-  private
-
-  def consent_enabled?
-    ActiveRecord::Type::Boolean.new.deserialize(params[:consent_enabled])
-  end
```

Comment:
We'll still need a way to enable and disable the feature: any petition template that does not have a gdpr consent section will need to follow the old flow, unless we deprecate all other templates.

Reply:
By the 25th all live templates need to be GDPR compilant. I talked to Tanya about adding a feature flag and we agreed it doesn't make sense, since all pages are going to have to be compilant in a couple weeks.